### ISSUE TITLE
The default module should be called `index` (not `main`)

### ISSUE THREAD

**Issue by gzm0** [scala-js/scala-js#4856]
I realized this when looking at https://github.com/gemini-hlsw/lucuma-core/pull/708#discussion_r1174045510

`index.js` / `index.ts` is special in the entire JS ecosystem as the "default" module (similar to `__init__.py` in python).

It is likely not worth it to change this now, but opening this anyways so we can have the discussion.

**Comment by sjrd** [comment]
More annoying than worth it or not, it's likely a breaking change. I don't see how we would change that.

**Comment by gzm0** [comment]
Yes, it is for sure a breaking change in practice. Maybe it goes on the Scala.js 2.x list? :P

**Comment by sjrd** [comment]
It's probably 2.x territory, yes :p
